_playwright-tests/auth.setup.ts

```diff
@@ -107,6 +108,25 @@ setup.describe('Setup Authentication States', async () => {
     await logout(page);
   });
 
+  setup('Authenticate no-subs user and save state', async ({ page }) => {
+    setup.skip(!process.env.RBAC, `Skipping as the RBAC environment variable isn't set to true.`);
```

Comment:
we don't need to authenticate this user unless we're running integration tests, so this auth step should be skipped if either `RBAC` or `INTEGRATION` is set to true, similar to the skip statement you have in the NoSubsUserTest

Reply:
\* _if either `RBAC` or `INTEGRATION` is set to **false**_

Reply:
OK, changes made, thank you for the reviews.